src/chainlit/frontend/src/components/organisms/chat/message/inlined.tsx

```diff
@@ -1,16 +1,16 @@
 import { Stack } from '@mui/material';
 
-import InlinedActionList from 'components/action/inlined';
-import InlinedAudioList from 'components/element/inlined/audios';
-import InlinedFileList from 'components/element/inlined/files';
-import InlinedImageList from 'components/element/inlined/images';
-import InlinedPDFList from 'components/element/inlined/pdfs';
-import InlinedTextList from 'components/element/inlined/texts';
-import InlinedVideoList from 'components/element/inlined/videos';
-
 import { IAction } from 'state/action';
 import { AllElements, ElementType, IElements } from 'state/element';
 
+import InlinedActionList from '../../../atoms/action/inlined';
```

Comment:
Great initiative but it looks live the imports were automatically refactored by your IDE. It would be nice to use absolute imports and not relative imports ie:

../../../atoms/element/inlined/audios -> components/atoms/element/inlined/audios

Reply:
It was intentional. I've experienced issues with absolute imports in the past.. But yes, I can do that if nothing breaks.